### ISSUE TITLE
I guess this is it.

### ISSUE THREAD

**Issue by darkfus10n** [kpwn/yalu102#472]
Well it seems to be that this is it. No more stable version. Stable as it gets. IPad Air 2 still very unstable but I must say a massive shout out to @kpwn and everyone else involved in this. At least we have a jailbreak. This is where it ends. Until........

**Comment by igobythisname** [comment]
i do appreciate all the effort that's gone into the iOS 10 jailbreak, but given the instability on the iPad Air 2, and now with Cydia Impactor 0.9.41, my 6SPlus is crashing to Safe Mode daily and other sorts of freezes and crashes, I am filled with regret over updating to iOS 10, when everything was running perfect on iOS 9, not to mention the liberty of Cydia Eraser.

**Comment by darkfus10n** [comment]
Well its not as bad as mine I guess..... @igobythisname

**Comment by darkfus10n** [comment]
I mean mine constantly restarts and re-enabling the jb takes ages. @igobythisname

**Comment by igobythisname** [comment]
are you referring to the iPad Air 2? Mine is very bad! I'm sitting here right now, must've hit 'go' about 20 times already trying to jailbreak.. it's one thing that it randomly hardly never jailbreaks, but it's another thing that just sitting there, holding, reading on the iPad, that the device just restarts or crashes to Safe Mode, and I have little to no tweaks/packages installed other than the initial Cydia setup packages, ugh!!!

**Comment by darkfus10n** [comment]
Yes I have iPad Air 2. Mine doesn't safe mode. It just randomly restarts @igobythisname

**Comment by jemmy87** [comment]
wtf

**Comment by Catzzye** [comment]
5S has some tiny hiccups here and there but it stable.

**Comment by ohmza** [comment]
For those who have troubles with iPad Air 2 and iPad Mini 4, I have already created the Pull Request that fixed the bug. Please go to #455 and feel free to report me any issues after testing.
